modules/management-console-agent/src/main/java/org/apache/ignite/internal/agent/utils/AgentUtils.java

```diff
@@ -214,14 +215,56 @@ public static void authorizeIfNeeded(IgniteSecurity security, String name, Secur
      * Quietly closes given processor ignoring possible checked exception.
      *
      * @param proc Process.
+     * @param log Logger.
      */
-    public static void quiteStop(GridProcessor proc) {
+    public static void stopProcessor(GridProcessor proc, IgniteLogger log) {
         if (proc != null) {
             try {
                 proc.stop(true);
             }
-            catch (Exception ignored) {
-                // No-op.
+            catch (Exception ex) {
+                U.warn(log, ex.getMessage(), ex);
```

Comment:
I would say, that errors reported during stopping are not that critical. I would change the logging level to DEBUG. What do you think?

Reply:
I disagree with you. `IgniteLogger` with logging level to DEBUG can't print stacktrace. I think we need stacktrace for debug.